pkg/hugo/frontmatter.go

```diff
@@ -61,6 +58,15 @@ func parseFrontMatter(r io.Reader) (*FrontMatter, error) {
 	if fm.Date, err = getContentDate(&cfm); err != nil {
 		return nil, err
 	}
+	if isArray := isArray(&cfm, fmAuthor); isArray {
+		if fm.Author, err = getFirstStringItem(&cfm, fmAuthor); err != nil {
+			return nil, err
+		}
+		return fm, nil
```

Comment:
Looks nice! Only the following small change and we should be good to merge.
I think It's better to enclose the next `if` block in the `else` and return the result only at the end. It could make a mistake when parsing a new item.